extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -43,6 +43,30 @@ public static FlywayDataSourceRuntimeConfig defaultConfig() {
     @ConfigItem
     public Optional<String> defaultSchema = Optional.empty();
 
+    /**
+     * The JDBC URL that Flyway will use to connect to the datasource.
+     * Falls back to the datasource URL if not specified.
+     */
+
```

Comment:
I saw in several pull requests of yours that you add a new line after the javadoc. Let's not do that for the next ones. Thanks!